src/app/my-info/(account)/wishlist/_components/WishCheckBox.tsx

```diff
@@ -0,0 +1,35 @@
+'use client';
+
+import classNames from 'classnames/bind';
+import { ReactNode } from 'react';
+import styles from './WishCheckBox.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface WishCheckBoxProps {
+  children?: ReactNode;
+  productId: number;
+  onChange: (id: number) => void;
+  isChecked: boolean;
+}
+
+export default function WishCheckBox({ children, productId, onChange, isChecked }: WishCheckBoxProps) {
+  const handleOnChange = () => {
+    onChange(productId);
+  };
+
+  return (
+    <>
+      <input
+        type='checkbox'
+        id={`select-item-${productId}`}
+        className={cn('select-item-input')}
+        checked={isChecked}
+        onChange={handleOnChange}
+      />
+      <label htmlFor={`select-item-${productId}`} className={cn('select-item-label')}>
+        {children}
+      </label>
+    </>
```

Comment:
```suggestion
    <label htmlFor={`select-item-${productId}`} className={cn('select-item-label')}>
      <input
        type='checkbox'
        id={`select-item-${productId}`}
        className={cn('select-item-input')}
        checked={isChecked}
        onChange={handleOnChange}
      />
        {children}
      </label>
```
이런식으로 쓰시면 프라그먼트가 없어도 될거같네요